src/CareTogether.Core/Permission.cs

```diff
@@ -42,9 +42,10 @@ public enum Permission
 
         AddEditDraftNotes = 180,
         DiscardDraftNotes = 181,
-        ApproveNotes = 182,
```

Comment:
We need to keep the integer values consistent for existing permissions, since _config.json_ files will otherwise give incorrect permissions for existing custom roles.